requirements.txt

```diff
@@ -1,3 +1,3 @@
 aiocoap==0.4.3
 DTLSSocket==0.1.12
-
+pydantic==1.9.0
```

Comment:
We should add it to `install_requires` in setup.py too, without the version limit.